[2023-12-12 04:10:11,229] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL.cargar_data manual__2023-12-12T04:09:33.823825+00:00 [queued]>
[2023-12-12 04:10:11,255] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL.cargar_data manual__2023-12-12T04:09:33.823825+00:00 [queued]>
[2023-12-12 04:10:11,256] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-12 04:10:11,257] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-12 04:10:11,257] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-12 04:10:11,291] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cargar_data> on 2023-12-12 04:09:33.823825+00:00
[2023-12-12 04:10:11,300] {standard_task_runner.py:52} INFO - Started process 480 to run task
[2023-12-12 04:10:11,309] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL', 'cargar_data', 'manual__2023-12-12T04:09:33.823825+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather.py', '--cfg-path', '/tmp/tmp7g2w4q1b', '--error-file', '/tmp/tmpje1pm3i2']
[2023-12-12 04:10:11,313] {standard_task_runner.py:80} INFO - Job 9: Subtask cargar_data
[2023-12-12 04:10:11,686] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL.cargar_data manual__2023-12-12T04:09:33.823825+00:00 [running]> on host 9ceec7b750bf
[2023-12-12 04:10:11,995] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-12 04:10:12,241] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL
AIRFLOW_CTX_TASK_ID=cargar_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-12T04:09:33.823825+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-12T04:09:33.823825+00:00
[2023-12-12 04:10:12,244] {logging_mixin.py:115} INFO - Cargando la data para la fecha: 2023-12-12 4
[2023-12-12 04:10:12,300] {logging_mixin.py:115} INFO - (1, 18)
[2023-12-12 04:10:12,372] {logging_mixin.py:115} INFO -                     dt          name  ...         sunrise_time          sunset_time
0  2023-12-12 01:05:08  Buenos Aires  ...  2023-12-12 05:34:15  2023-12-12 20:00:11

[1 rows x 18 columns]
[2023-12-12 04:10:14,473] {logging_mixin.py:115} INFO - Carga completada en Redshift!
[2023-12-12 04:10:14,475] {python.py:173} INFO - Done. Returned value was: None
[2023-12-12 04:10:14,527] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL, task_id=cargar_data, execution_date=20231212T040933, start_date=20231212T041011, end_date=20231212T041014
[2023-12-12 04:10:14,585] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-12 04:10:14,774] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
